### ISSUE TITLE
Biopax API/pyjnius won't run on Travis, causing tests to time out and fail

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#32]
Typical output from Travis:

```
$ nosetests --with-doctest
EFSLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

The build has been terminated
```

**Comment by bgyori** [comment]
This has happened with pyjnius in other settings too, independent of BioPAX. It is not clear yet what is causing it to hang in certain situations.

**Comment by johnbachman** [comment]
OK, totally weird, but this seems to have spontaneously resolved, at least as of commit 62c79dba8fa5 (build 79 on Travis). We'll just have to wait and see if it happens again, I guess.

**Comment by bgyori** [comment]
This is something that happens sometimes and not other times. I have no
idea what determines whether it times out or not. I suspect it might be a
problem with jnius itself, not Pathway Commons.
